OpenOversight/app/templates/list_officer.html

```diff
@@ -98,13 +98,15 @@ <h3 class="panel-title accordion-toggle">Rank</h3>
             <h3 class="panel-title accordion-toggle">Unit</h3>
           </div>
           <div class="collapse" id="filter-unit">
-            <div class="panel-body">
+            <div class="panel-body panel-body-long">
               <div class="form-group">
-                <select class="form-control" name="unit">
```

Comment:
`select` actually has a `multiple` attribute that might be perfect for this instead of a checkbox and extra CSS: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/select#attr-multiple

IIRC it will render it as scrollable text box of a fixed height. Though might not be as intuitive to some folks how to select multiple (you have to control + click to select more than one... though most folks with moderate computer literacy will have learned that trick I think? Not sure)